src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -92,7 +92,32 @@ def read_qlats(forcing_parameters, segment_index, nexus_to_downstream_flowpath_d
     qlat_df = pd.concat( (nexuses_flows_df.loc[int(k)].rename(index={int(k):v})
         for k,v in nexus_to_downstream_flowpath_dict.items() ), axis=1
         ).T
-    
+
+    #For a terminal nexus, we want to include the lateral flow from the catchment contributing to that nexus
+    #one way to do that is to cheat and put that lateral flow at the upstream...this is probably the simplest way
+    #right now.  The other is to create a virtual channel segment downstream to "route" i.e accumulate into
+    #but it isn't clear right now how to do that with flow/velocity/depth requirements
+    #find the terminal nodes
+    for tnx, up in terminal_links.items():
+        print(up, tnx)
```

Comment:
Is this print supposed to be here?

Reply:
Only if you like non-contextual debugging information.